recipes-devtools/clang/compiler-rt_git.bb

```diff
@@ -61,7 +61,7 @@ EXTRA_OECMAKE += "-DCOMPILER_RT_STANDALONE_BUILD=OFF \
                   -DCOMPILER_RT_BUILD_SANITIZERS=OFF \
                   -DCOMPILER_RT_BUILD_MEMPROF=OFF \
                   -DCOMPILER_RT_BUILD_LIBFUZZER=OFF \
-                  -DCOMPILER_RT_BUILD_PROFILE=OFF \
+                  -DCOMPILER_RT_BUILD_PROFILE=ON\
```

Comment:
maybe make it a packageconfig ?

Reply:
Good idea.  I have updated PR to do it like that.